scripts/jest/preprocessor.js

```diff
@@ -52,13 +49,17 @@ module.exports = {
       !filePath.match(/\/node_modules\//) &&
       !filePath.match(/\/third_party\//)
     ) {
-      return babel.transform(
+      var code =
+      babel.transform(
         src,
         Object.assign(
           {filename: path.relative(process.cwd(), filePath)},
           babelOptions
         )
       ).code;
+      if (/ReactElementValidator\-test/.test(filePath))
+        console.log(code);
```

Comment:
oooooops, this is how I try to debug to see if the transform did something weird. This would obviously go away.